flutter/lib/button.dart

```diff
@@ -21,6 +21,7 @@ class Button extends StatefulWidget {
       text: text,
       onPressed: onPressed,
       color: kSecondaryColor,
+      buttonHeight: null,
```

Comment:
```dart
factory Button.secondary({
    Key? key,
    required String text,
    VoidCallback? onPressed,
    double? buttonHeight,
  }) {
    return Button(
      key: key,
      text: text,
      onPressed: onPressed,
      color: kSecondaryColor,
      buttonHeight: buttonHeight,
```

Reply:
fixed according to the suggestion for .secondary and tertiary